nixos/modules/services/networking/networkmanager.nix

```diff
@@ -454,8 +436,16 @@ in
   imports = [
     (mkRenamedOptionModule
       [ "networking" "networkmanager" "packages" ]
-      [ "networking" "networkmanager" "plugins" ])
-    (mkRenamedOptionModule [ "networking" "networkmanager" "useDnsmasq" ] [ "networking" "networkmanager" "dns" ])
+      [ "networking" "networkmanager" "plugins" ]
+    )
+    (mkRenamedOptionModule
+      [ "networking" "networkmanager" "useDnsmasq" ]
+      [ "networking" "networkmanager" "dns" ]
+    )
+    (mkRenamedOptionModule
```

Comment:
hmmm i don't think it's quite a rename, this is a semantic change to be an attr set that should have a nicer message and probably stop evaluation saying how to fix it.

Reply:
Can you share a example of how to do that or point me to the correct function?

Reply:
`mkRemovedOptionModule` with a good description?
My expectation would be otherwise that I can rename the option in my config and call it a day.

That said, perhaps it's good to keep `extraConfig` for one more release?

Reply:
> That said, perhaps it's good to keep extraConfig for one more release?

I can almost certainly guarantee you that everyone will just forgot about it until then, so I would either do it now or leave it as is, but I don't have hard opinions on this.

> mkRemovedOptionModule with a good description?

sounds good to me.

Reply:
> I can almost certainly guarantee you that everyone will just forgot about it until then

I mean, it should give an annoying warning to make sure that this won't happen ;)

Reply:
I changed it to `mkRemovedOptionModule` (not sure if the markdown works there).
> > I can almost certainly guarantee you that everyone will just forgot about it until then

> I mean, it should give an annoying warning to make sure that this won't happen ;)

I meant if we do not rename extraConfig to settings that no body will remember to rename it later.
But I probably just misunderstood: "That said, perhaps it's good to keep extraConfig for one more release?"